packages/api/portfolioDrafts/funding/getFundingStep.test.ts

```diff
@@ -0,0 +1,91 @@
+import { Context } from "aws-lambda";
+import { ApiSuccessResponse, ErrorStatusCode, OtherErrorResponse, SuccessStatusCode } from "../../utils/response";
+import { handler } from "./getFundingStep";
+import { DynamoDBDocumentClient, GetCommand, GetCommandOutput } from "@aws-sdk/lib-dynamodb";
+import { FundingStep, ValidationMessage } from "../../models/FundingStep";
+import { mockClient } from "aws-sdk-client-mock";
+import { v4 as uuidv4 } from "uuid";
+import { DATABASE_ERROR, NO_SUCH_FUNDING_STEP, NO_SUCH_PORTFOLIO_DRAFT } from "../../utils/errors";
+import { ApiGatewayEventParsed } from "../../utils/eventHandlingTool";
+
+const ddbMock = mockClient(DynamoDBDocumentClient);
+beforeEach(() => {
+  ddbMock.reset();
+});
+
+const emptyRequest: ApiGatewayEventParsed<FundingStep> = {} as any;
+const mockFundingStep: FundingStep = {
+  task_orders: [
+    {
+      task_order_file: {
+        name: "dummy.pdf",
+        id: "134dfe66-f9fc-4f2e-9f39-b6f7e53e543a",
+      },
+      clins: [
+        {
+          idiq_clin: "IDIQ CLIN 0002 Classified IaaS/PaaS",
+          clin_number: "0091",
+          pop_start_date: "2019-10-05",
+          pop_end_date: "2021-12-28",
+          total_clin_value: 9131,
+          obligated_funds: 1477,
+        },
+      ],
+      task_order_number: "8787878787878999",
+    },
+  ],
+};
```

Comment:
I know PR #375 got merged yesterday with the additional `common*MockData` files, so I would suggest adding this to the `commonFundingMockData.ts` with the *good data* after a rebase with those changes.